tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = py35, py36, docs
+
+[travis]
+python =
+    3.6: py36
+    3.5: py35
+
+[testenv]
+passenv = TOXENV CI TRAVIS TRAVIS_*
+extras = testing
+commands =
+    pytest --cov=flask_restalchemy
```

Comment:
@nicoddemus, please check if this is the right way to do it since `--cov={envsitepackage}/flask-restalchemy` gives 0% coverage

Reply:
many thks for that